StdAfx.h

```diff
@@ -62,9 +62,14 @@
 #pragma comment(lib, "rlib/libcef_dll_wrapper.lib")
 #endif
 
+#pragma warning(push, 0)

```

Comment:
Adding this `#pragma` is the main change of this file.
Other changes are made by auto formatter.